tests/unit/type/list.tcl

```diff
@@ -339,6 +555,48 @@ start_server {
         assert_equal {foo} [r lrange target2 0 -1]
     }
 
+    test "BRPOPRPUSH with multiple blocked clients" {
```

Comment:
i think we can trim these 3, don't add real value

Reply:
Done.